Userland/Libraries/LibWeb/Layout/Node.cpp

```diff
@@ -666,6 +667,15 @@ JS::NonnullGCPtr<NodeWithStyle> NodeWithStyle::create_anonymous_wrapper() const
     return *wrapper;
 }
 
+void NodeWithStyle::reset_table_box_computed_values_used_by_wrapper_to_init_values()
+{
+    VERIFY(is<TableBox>(*this));
```

Comment:
Can we put this in `TableBox` instead? It's not relevant for any other class, and then we don't need this check.

Then we can simplify the name a bit as well, e.g:
```cpp
void TableBox::reset_computed_values_used_by_wrapper_to_initial_values()
```

Reply:
as I see it has to be in `NodeWithStyle` because it is the only place where computed values can be mutated unless we move `m_computed_values` from private members.

Reply:
Ehh yeah okay fine :)